go/apps/api/openapi/spec/common/EmptyResponse.yaml

```diff
@@ -1,4 +1,5 @@
 type: object
 properties: {}
 additionalProperties: false
-description: Empty response object by design. A successful response indicates this operation was successfully executed.
+description: Empty response object by design. A successful response indicates the key was deleted successfully.
```

Comment:
This is not only used for key deletion and should reain its >  this operation was successfully executed.